mindmeld/models/model.py

```diff
@@ -1295,6 +1295,9 @@ def decode(self, tags_by_example, **kwargs):
             list: A list of decoded labels per query
         """
         # TODO: support decoding multiple queries at once
+        if not hasattr(self, "system_entity_recognizer"):
+            # app built with older version of MM (< 4.3) does not save label encoder with sys recognizer
+            self.system_entity_recognizer = SystemEntityRecognizer.get_instance()
```

Comment:
Why do it in `decode` method instead of in `__init__ ` for example?

Reply:
The initialization is already inside the `__init__` method; I suspect `__init__` is not being called for deserialized objects.

https://github.com/cisco/mindmeld/pull/209/files#diff-67b2786b2832b1a5d1ce4b306d7f2b67L1261